balrogscript/script.py

```diff
@@ -231,7 +232,15 @@ def setup_config(config_path):
             usage()
         config_path = sys.argv[1]
 
-    config = load_config(config_path)
+    data_dir = os.path.join(os.path.dirname(__file__), 'data')
+    config = {
+        'schema_files': {
+            'submit-locale': os.path.join(data_dir, 'balrog_submit-locale_schema.json'),
+            'submit-toplevel': os.path.join(data_dir, 'balrog_submit-toplevel_schema.json'),
+            'schedule': os.path.join(data_dir, 'balrog_schedule_schema.json')
+        },
+    }
+    config.update(load_config(config_path))
     return config
```

Comment:
Nit: Add a TODO to say we should probably use https://github.com/mozilla-releng/scriptworker/blob/a86e056c8526238d690bd31a323d7adec6d13cb7/scriptworker/client.py#L136

Reply:
I'll note that we aren't currently using it because balrogscript is on python 2.7. (We need to port [balrogclient](https://github.com/mozilla/balrog/tree/master/client) and [balrogscript.submitter](https://github.com/mozilla-releng/balrogscript/tree/master/balrogscript/submitter) to python 3.)

Reply:
Oh yeah, that rings a bell. There's actually an issue on file https://github.com/mozilla-releng/balrogscript/issues/7#issuecomment-374537921. tom++ for remembering this!

Reply:
Based on that issue, it looks like balrogclient already supports py3, so all the code that needs to change is in this repo.